src/main/java/tictactoe/BoardConverter.java

```diff
@@ -11,38 +13,87 @@ public String toString(Marks[][] board) {
 
         for (Marks[] columns : board) {
             for (Marks mark : columns) {
-                str.append(createLine(idSpot, mark));
+                str.append(createSpot(idSpot, mark));
                 idSpot++;
             }
 
             if (++idLine != board.length) {
-                str.append("-----------\n");
+                str.append(repeatElement(totalCharBoardWidth(board), INTER_LINE));
+                str.append(NEW_LINE);
             }
         }
 
         return str.toString();
     }
 
-    private String createLine(int idSpot, Marks mark) {
-        return " " + markToDisplay(mark, idSpot) + rightEdgeSpot(idSpot);
+    private String createSpot(int idSpot, Marks mark) {
```

Comment:
Similar like before, I don't know what `idSpot` means when reading the method definition.